src/traces/parcoords/calc.js

```diff
@@ -10,16 +10,9 @@
 
 var hasColorscale = require('../../components/colorscale/has_colorscale');
 var calcColorscale = require('../../components/colorscale/calc');
-var Lib = require('../../lib');
 
 
 module.exports = function calc(gd, trace) {
-    var cs = !!trace.line.colorscale && Lib.isArray(trace.line.color);
-    var color = cs ? trace.line.color : Array.apply(0, Array(trace.dimensions.reduce(function(p, n) {return Math.max(p, n.values.length);}, 0))).map(function() {return 0.5;});
-    var cscale = cs ? trace.line.colorscale : [[0, trace.line.color], [1, trace.line.color]];
-
```

Comment:
I just moved this block unchanged from here, where it'd be persisted, to parcoords.js below, as this calculation is now considered implementation detail, unexposed on data/fullData